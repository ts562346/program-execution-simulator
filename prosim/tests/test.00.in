test00 2
DOOP 2
HALT

