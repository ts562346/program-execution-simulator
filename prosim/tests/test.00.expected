00000: DOOP
00002: HALT

DOOP count : 1
DOOP time  : 2
BLOCK count: 0
BLOCK time : 0
